Tribler/Core/CacheDB/sqlitecachedb.py

```diff
@@ -13,17 +13,14 @@
 from Tribler.Core.CacheDB.db_versions import LATEST_DB_VERSION
 
 
-DB_SCRIPT_NAME = u"schema_sdb_v%s.sql" % str(LATEST_DB_VERSION)
-
-DB_FILE_NAME = u"tribler.sdb"
 DB_DIR_NAME = u"sqlite"
+DB_FILE_NAME = u"tribler.sdb"
 DB_FILE_RELATIVE_PATH = os.path.join(DB_DIR_NAME, DB_FILE_NAME)
-
+DB_SCRIPT_NAME = u"schema_sdb_v%s.sql" % str(LATEST_DB_VERSION)
+DB_SCRIPT_RELATIVE_PATH = os.path.join("Core/CacheDB/", DB_SCRIPT_NAME)
```

Comment:
@devos50 How should I do this? Retrieving from `__file__` seems dirty. Note that the previous location of the script (in the state directory) was hard coded as well.